test/TerminalLogger.jl

```diff
@@ -1,7 +1,56 @@
 import TerminalLoggers.default_metafmt
+import TerminalLoggers.format_message
```

Comment:
It's not really about this PR, but why not

```suggestion
using TerminalLoggers: default_metafmt, format_message
```

?  I find it nice that you don't need to worry about accidentally extending those functions.